projects/js-packages/components/components/icon-tooltip/index.tsx

```diff
@@ -57,14 +58,11 @@ const IconTooltip: React.FC< IconTooltipProps > = ( {
 	};
 
 	const args = {
-		iconCode,
 		// To be compatible with deprecating prop `position`.
 		position: placementsToPositions( placement ),
 		placement,
 		animate,
 		noArrow: false,
-		resize: false,
-		flip: false,
```

Comment:
Yes! They are intentionally set to override the default value to the `Popover` component, which keeps the panel displaying fixed for usage along with the icon.

Reply:
Yeah, I deleted these because they caused some JS errors in the console:

<img width="914" alt="CleanShot 2022-09-20 at 09 42 05@2x" src="https://user-images.githubusercontent.com/36671565/191197377-39eec819-2c07-4365-b896-5d6901de9ac4.png">

I followed the Popover declaration to the interface, and didn't see these attributes there, so I thought they changed, but I was wrong then sorry!

What do you think about these errors?

Reply:
It looks like that's this error: https://styled-components.com/docs/faqs#why-am-i-getting-html-attribute-warnings

I can reproduce it. Those props were [only added recently](https://github.com/WordPress/gutenberg/commit/63d89fb7f5973aa7afb617ba33b85d16788aceaf) to the `Popover` component.

It appears to be a mismatch between the latest version of the component and the version of `@wordpress/components` we're using in the monorepo. These props don't exist on the component yet in the version we're using, so they're get passed directly to the underlying HTML element (something like `<div flip={ false } resize={ false }>`.

It's probably fine to just ignore this warning for now, it'll go away when the dependency gets updated.